### DISCUSSION TITLE
too many timer controller

### DISCUSSION THREAD

**Discussion by fengzk-edgeray** [taosdata/TDengine#7598] [Q&A]
![image](https://user-images.githubusercontent.com/89452329/130888343-daef6e9d-84ad-4fdf-a7d4-6856d05ce17b.png)
I build 3 nodes tdengine cluster, I init a connection to SecondEndpoint(SecondEp),but it happen to break errors.
background: 
tdengine version:2.0.14.0 
node1(secondEp) node2 node3(FirstEp) 
go driver , taoSql connection to root:pass@tcp:node1:6030:/{db}

**Comment by sangshuduo** [comment]
2.0.14.0 is pretty old. Please try the latest version and let's know if the problem is still exists.
